app/uk/gov/hmrc/apidocumentation/models/Documentation.scala

```diff
@@ -131,66 +129,53 @@ object APIAccess {
   )
 }
 
-case class APIDefinition(
-    serviceName: String,
-    name: String,
-    description: String,
-    context: String,
-    requiresTrust: Option[Boolean],
-    isTestSupport: Option[Boolean],
-    versions: Seq[APIVersion],
-    categories: Option[Seq[APICategory]] = None
-  ) extends Documentation {
+case class WrappedApiDefinition(definition: ApiDefinition) extends Documentation {
+  override val name: String                         = definition.name
+  override val context: String                      = definition.context.value
+  override val categories: Option[Seq[ApiCategory]] = Some(definition.categories)
```

Comment:
Can we remove the option seq and just end up with a list??

Reply:
I didn't want to change the base Trait as it is used in a bunch of places. With the logic to fall back to the json file to get it if they don't have categories for some APIs so while we're sure APIDef have categories the roadmap/serviceguides/xml might not